src/core_init_atmosphere/mpas_geotile_manager.F

```diff
@@ -686,12 +686,15 @@ end function mpas_geotile_mgr_gen_tile_name
     !> \author  Miles A. Curry
     !> \date    02/20/2020
     !> \details
-    !> Given a tile, translate the relative tile coordinates, i, j, of that
-    !> tile to a latitude and longitude coordinate. Upon success, lat, lon
-    !> will be in the range of -1/2 * pi to 1/2 * pi and 0 to 2.0 * pi, respectively.
+    !> Given a tile, translate the relative tile coordinates, i, j, of that tile to a
+    !> latitude and longitude coordinate. Upon success, lat, lon will be in the range
+    !> of -1/2 * pi to 1/2 * pi and 0 to 2.0 * pi, respectively. If supersample_fac is
+    !> present, then the tile will be subdivided into 2 * supersample_fac pixels
+    !> (supersample_fac for both the north-south and east-west directions).
```

Comment:
@MiCurry As we had discussed, I think, at a minimum, `2 * supersample_fac` should be `supersample_fac ^ 2`. But, the phrase "the tile will be subdivided into ..." might suggest to some readers that it is the `mpas_geotile_mgr_to_latlon` routine that is doing the subdividing, when in fact it is the calling code that has chosen to sub-divide a tile, and it's this routine that simply needs to take that into account when computing lat/lon coordinates.

Reply:
@mgduda how does this sound:

> Given a tile, translate the relative tile coordinates, i, j, of that tile to a corresponding latitude and longitude coordinate.
> 
> If supersample_fac is present, the latitude, longitude coordinates returned will represent pixel centers of a tile that has been subdivided into 2 ^ supersample_fac pixels. Therefore, the calling code will need to pass in supersampled i, j coordinates as each i, j coordinate will represent a divided tile with supersample_fac * (tile_nx/tile_ny + tile_bdr) pixels.
> 
> Upon success, lat and lon will be in the range of -1/2 * pi to 1/2 pi and 0 to 2.0 * pi, respectively.

Reply:
@mgduda I have just pushed an update with changes to the above. Let me know if you think it still needs more work. Thanks for the feedback on this.

Reply:
The updated version looks good.